src/models/dto/dissolutionCreateResponse.ts

```diff
@@ -0,0 +1,3 @@
+export interface DissolutionCreateResponse {
+  application_reference_number: string
```

Comment:
don't forget about the links too, even if they aren't used in the frontend we may as well keep the api models consistent